GETTING_STARTED.md

```diff
@@ -0,0 +1,415 @@
+
+## Hello-world example
+
+We'll be using a very simple yaml files example which you can find [here](./examples/hello-world-yaml/README.md). Additionaly, we'll discuss every single step performed to make sure that everything regarding Eunomia operation is clear.
+
+### Create a new namespace
+
+First, we need to create a namespace for our resources to be deployed in. This is for the sake of separation and keeping order:
+
+```shell
+kubectl create namespace eunomia-hello-world-yaml-demo
+```
+
+Earlier we deployed Eunomia with the Helm Chart with default settings - with such set, Eunomia watches all namespaces in the cluster for GitOpsConfigs.
+
+### Create a new service account
+
+We need the service account to provide access rights for all pods to be created without problems. The service account's name is `eunomia-runner-yaml`. It will be bound to `ClusterRole admin` to not complicate the example too much.
+
+```shell
+kubectl apply -f examples/hello-world-yaml/eunomia-runner-sa.yaml -n eunomia-hello-world-yaml-demo
+```
+
+### Deploy the GitOpsConfig Custom Resource for the hello-world application
+
+Eunomia is a Kubernetes native application ([an operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)). Its [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) being watched for is a GitOpsConfig.
+
+
+This is [the one](https://github.com/KohlsTechnology/eunomia/blob/master/examples/hello-world-yaml/cr/hello-world-cr1.yaml) which will be applied:
+
+```yaml
+apiVersion: eunomia.kohls.io/v1alpha1
+kind: GitOpsConfig
+metadata:
+  name: hello-world-yaml
+spec:
+  templateSource:
+    uri: https://github.com/KohlsTechnology/eunomia
+    ref: master
+    contextDir: examples/hello-world-yaml/template1
+  parameterSource:
+    ref: master
+    contextDir: examples/hello-world-yaml/parameters
+  triggers:
+  - type: Change
+  serviceAccountRef: eunomia-runner-yaml
+  templateProcessorImage: quay.io/kohlstechnology/eunomia-base:latest
+  resourceHandlingMode: Apply
+  resourceDeletionMode: Delete
+```
+Things to pay attention to:
+|  field name  |  description |
+|:---|:---|
+|  `kind`  |  set to `GitOpsConfig` - no Kubernetes application but Eunomia will be caring for it  |
+|  `templateSource` section  |  it defines the source of the templates  |
+|  - `uri`  |  repository where the templates are taken from (pay heed to the fact that this is [the official Eunomia repo](https://github.com/KohlsTechnology/eunomia), not your personal fork!)  |
+|  - `ref`  |  branch on the remote repository  |
+|  - `contextDir`  |  directory where templates files will be taken from; in our example it's [this one](https://github.com/KohlsTechnology/eunomia/tree/master/examples/hello-world-yaml/template1) - you can find here a [single manifest file](https://github.com/KohlsTechnology/eunomia/blob/master/examples/hello-world-yaml/template1/hello-world.yaml) with a Deployment and a Service inside  |
```

Comment:
similar as above: please don't use `this one` as link text, instead make the text understandable even if link targets were missing